src/runtime/contrib/cublas/cublas_utils.h

```diff
@@ -80,7 +80,13 @@ struct CuBlasThreadEntry {
 struct CuBlasLtThreadEntry {
   CuBlasLtThreadEntry();
   ~CuBlasLtThreadEntry();
+
   cublasLtHandle_t handle{nullptr};
+  cublasLtMatmulPreference_t matmul_pref_desc{nullptr};
+  void* workspace_ptr{nullptr};
+  // 32MB workspace.
+  constexpr const static size_t workspace_size = 33554432;
```

Comment:
What's special about this particular size?

Reply:
it's recommended value for Hopper. for previous archs, it should be 4MB

Reply:
Where is that documented?

Reply:
https://docs.nvidia.com/cuda/cublas/index.html#cublassetworkspace

Reply:
Thanks! Just updated with the source.